QEfficient/generation/text_generation_inference.py

```diff
@@ -199,10 +199,13 @@ def fix_prompts(prompt: List[str], batch_size: int, full_batch_size: int = None)
 
 def read_prompts_txt_file(prompts_txt_file_path: str):
     prompt = []
-    with open(prompts_txt_file_path, "r") as file:
-        for line in file:
-            prompt.append(line.strip())
-    return prompt
+    try:
+        with open(prompts_txt_file_path, "r") as file:
+            for line in file:
+                prompt.append(line.strip())
+        return prompt
+    except OSError:
+        print("Error: File not found.")
```

Comment:
why do you need to change this?

Reply:
As per Rishin, it is better to include a try except block when performing any I/O operations. Therefore, it's a good to have change.

Reply:
If the file is not there it will raise `FIleNotFoundError`, which is expected.
Also, in case of `OSError`, we should stop execution, instead of printing and then moving on.
I think the old code is okay.

Reply:
We should be raising FIleNotFoundError exception instead of OSError for if file not available.